### ISSUE TITLE
Some of the documentation contains unnecesarily gendered pronouns

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6857]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#6857|
|Type|defect|
|Created|2013-11-29 22:34:39Z|
|Branch|https://github.com/twisted/twisted/tree/they-6857|

This can make the documentation feel hostile. See: thoughtstreams.io/glyph/they-lives/

Attachments:

* [t6857.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a05/a05cb173c1ba1c6ae30bcf5edbd5d5e19566e764/90e5b4fbe5ee8959040b3fced55b965cd2ef9528.diff) (9345 bytes) - added by Alex on 2013-11-29 22:35:04Z - 
* [t6857.2.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a05/a05cb173c1ba1c6ae30bcf5edbd5d5e19566e764/d684614a3876ad4df2c032894f64fd623920e3d5.diff) (3378 bytes) - added by Alex on 2013-11-29 22:48:23Z - Previous patch accidentally also deleted portmap.c. While that is surely a meritrious cause, perhaps for another issue.

<details><summary>Searchable metadata</summary>

```
trac-id__6857 6857
type__defect defect
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__branches_they_6857 branches/they-6857
branch_author__glyph glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1385764479000000 1385764479000000
changetime__1385766787000000 1385766787000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

(In [40872]) Branching to 'they-6857'

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph commented|
|-|-|

(In [40873]) Commit t6857.2.diff re [#6857](https://github.com/twisted/twisted/issues/6857)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

Documentation build is clean and no other test runs completed thus far seem unduly troubled.  I'll land this.

However, we don't have any documentation or policy in place addressing future documentation.  Please file a separate ticket for updating `doc/core/development/policy/doc-standard.xhtml`.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph set status to `closed`|
|-|-|

(In [40875]) Remove some unnecessarily gendered language from the documentation.

Author: Alex
Reviewer: glyph
Fixes: [#6857](https://github.com/twisted/twisted/issues/6857)

Both "she" and "he" should be "they" when referring to an indeterminate
hypothetical programmer.
